packages/gatsby-plugin-gatsby-cloud/src/utils/trackEvent.js

```diff
@@ -0,0 +1,36 @@
+const pkgjson = require(`../package.json`)
```

Comment:
Use importing on the web as it will trigger webpack  JSON module logic.
```suggestion
import pkgJSON from "../package.json"
```